redwoods-insurance-salesforce/package.json

```diff
@@ -6,8 +6,8 @@
     "scripts": {
         "lint": "npm run lint:lwc",
         "lint:lwc": "eslint **/lwc/**",
-        "prettier": "prettier --write '**/*.{cmp,component,css,html,js,json,md,page,yaml,yml,cls,trigger}'",
-        "prettier:verify": "prettier --list-different '**/*.{html,js,json,yaml,yml,md,cmp,page,component,cls,trigger}'",
+        "prettier": "prettier --write '**/*.{xml,cmp,component,css,html,js,json,md,page,yaml,yml,cls,trigger}'",
+        "prettier:verify": "prettier --list-different '**/*.{xml,html,js,json,yaml,yml,md,cmp,page,component,cls,trigger}'",
```

Comment:
File extension parameters in alphabetical order please.

Reply:
Just addressed.